src/Misc/externals.sh

```diff
@@ -127,6 +127,10 @@ function acquireExternalTool() {
     fi
 }
 
+# Download the external tools for all platforms
+acquireExternalTool "$CONTAINER_URL/NuGet/3.5.1-beta1-1621/NuGet.zip" nuget
+acquireExternalTool "$CONTAINER_URL/CredentialProvider.TeamBuild/15.104.25518.0/CredentialProvider.TeamBuild.zip" nuget
```

Comment:
OK.  You'll have to provide instructions at a minimum for what they need to update on those distros for it to work.   The task readme should point to a kb link like our other tasks with those instructions.